content/en/docs/getting-started/_index.html

```diff
@@ -0,0 +1,266 @@
+      </div>
+      
+        Install the `jx` binary
+      
+{{<highlight sh>}}
+scoop install jx
+{{</highlight>}}
+      
+        Upgrade the `jx` binary
+      
+{{<highlight sh>}}
+scoop update jx
+{{</highlight>}}
+        </div>
+      </div>
+    </div>
+  </div>    
+
+  <p>&nbsp;</p>
+    <div style="row">
+      <div class="col col-lg-12">
+      <h2>Step 2: Provision your Kubernetes Cluster</h2>
+          
+      <p>Provisioning your Kubernetes cluster is easy. Jenkins X does not really care how you provision your cluster, however there
+          are many resources that are provisioned, so we recommend using the Terraform Modules we've made available</p>
+          
+      <ul class="nav nav-tabs" id="provision-cluster-tabs">
+        <li class="nav-item">
+          <a class="nav-link active" data-toggle="tab" href="#gcp">GCP</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link disabled" data-toggle="tab" href="#aws">AWS (coming soon)</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link disabled" data-toggle="tab" href="#azure">Azure (coming soon)</a>
+        </li>
+      </ul>
+      <div id="provision-cluster-tabs" class="tab-content">
+        <div class="tab-pane fade show active" id="gcp">
+          <div class="row">
+            <div class="col-sm-4">
+                  <a href="/docs/getting-started/setup/create-cluster/google/"><img src="https://www.pulumi.com/logos/tech/gcp.svg"/></i></a>
+            </div>
+          </div>
+          <h4>Use GKE Terraform Module</h4>
+          <p>1 . Create a <strong>main.tf</strong> file and add the code snippet below.</p>
+{{<highlight javascript>}}
+module "jx" {
+  source  = "jenkins-x/jx/google"
+  version = "1.2.1"
+  # override default parameter values here
+}
+{{</highlight>}}
+
+          <p>This setup uses defaults parameter values.  You can modify them, <a href="/docs/getting-started/setup/create-cluster/gke/#inputs">more details here.</a></p>
+        
+          <p>2 . Provision the cluster using Terraform. The output of this command will include a file <strong>jx-requirements.yml</strong> which<br/>
+          you will use in <strong>Step 3: Install Jenkins X on your cluster</strong> </p>
+{{<highlight bash>}}
+$ terraform init && terraform apply
+{{</highlight>}}
+
+        </div>
+        <div class="tab-pane fade" id="aws">
+        <div class="row">
+          <div class="col-sm-4">
+                <a href="/docs/getting-started/setup/create-cluster/amazon/"> <img src="https://www.pulumi.com/logos/tech/aws.svg" class="mx-auto d-block"/></i></a>
+          </div>
+        </div>
+        <h4>Use the AWS Terraform Module</h4>
+        <p>1 . Create a <strong>main.tf</strong> file and add the code snippet below.</p>
```

Comment:
"In an empty directory create a <strong>main.tf</strong> file and add the code snippet below."